charts/osdfir-infrastructure/charts/openrelik/templates/worker-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+{{- range .Values.workers }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $.Release.Name }}-{{ .name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: worker
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: worker
+    spec:
+      hostNetwork: false
+      hostIPC: false
+      automountServiceAccountToken: false
+      containers:
+        - name: worker
+          image: {{ .image | quote }}
+          imagePullPolicy: "IfNotPresent"
+          command: ["/bin/sh", "-c", "{{ .command }}"]
+          env:
+          {{- if and (eq .name "openrelik-worker-timesketch") $.Values.global.timesketch.enabled }}
```

Comment:
I think these env values should be moved to values.yaml so we can define these per worker.

Reply:
Because how we setup the integration between the tools it needs to be able to reference and render the release name for Timesketch integration. I at least added a way to add env variables for the rest of the workers though. We could look into another maybe better way for references between tools in the future (as lower priority)

Reply:
ack